lib/galaxy/workflow/modules.py

```diff
@@ -1244,27 +1256,19 @@ def load_module_sections(trans):
                 "title": "Input Dataset Collection",
                 "description": "Input dataset collection"
             },
+            {
+                "name": "pause",
+                "title": "Pause Workflow for Dataset Review",
+                "description": "Pause for Review"
+            },
+            {
+                "name": "parameter_input",
+                "title": "Parameter Input",
+                "description": "Simple inputs used for workflow logic"
+            },
         ],
     }
 
-    if trans.app.config.enable_beta_workflow_modules:
```

Comment:
My two cents is that I'd keep the option and just promote parameter_input to an un-beta parameter.